### ISSUE TITLE
After #2123 cache plugins are not working

### ISSUE THREAD

**Issue by astappiev** [roots/sage#2133]
## Submit a feature request or bug report

- [x ] I've read the [guidelines for Contributing to Roots Projects](https://github.com/roots/guidelines/blob/master/CONTRIBUTING.md)
- [ ] This is a feature request
- [x ] This is a bug report
- [x ] This request isn't a duplicate of an [existing issue](https://github.com/roots/sage/issues)
- [x ] I've read the [docs](https://roots.io/sage/docs) and [NPM Debugging Guidelines post](https://discourse.roots.io/t/npm-debugging-guidelines-failed-npm-install-bower-install-or-gulp-build-read-this/3060) and followed them (if applicable)
- [x ] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/c/sage) forums

---

**What is the current behavior?**

After applying changes from #2123, the cache plugin brokes website. All new caches do not contain content (only header section). So, when user request page, the white blank page is shown.

**What is the expected or desired behavior?**

As before, cached version should be same as non-cached.

---

## Bug report

**Please provide steps to reproduce, including full log output:**

1. Apply changes from #2123 
2. Install a cache plugin (I used [this one](https://wordpress.org/plugins/cache-enabler/), but also tried some another with same results)
3. Enable it
4. Open page (it will be loaded correctly, as no cache exists)
5. Reload it and instead of the expected page, you will see the blank page (as it will use a cached version of the page which is broken).

**Please describe your local environment:**

WordPress version: 4.9.8
